components/ProfileModal/Profile-Modal.tsx

```
@@ -103,45 +108,128 @@ const ProfileBottomSheet: React.FC<ProfileBottomSheetProps> = ({
+                      .map((item, index) => (
+                        <Image
+                          key={`top-${index}`}
+                          source={{ uri: item.url }}
+                          style={{
+                            width: 100,
+                            height: 100,
+                            borderRadius: 10,
+                            marginRight: 10,
+                          }}
+                        />
+                      ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)